game.js

```diff
@@ -55,6 +55,17 @@ class Game {
 		this.pauseUntilMilliSecond = millis() + ms
 	}
 
+  // General setup
+  setup(phrases){
```

Comment:
What's the purpose of the this 'general setup'?

Reply:
Did you merge the latest changes from the `main` branch into this branch yet?  I remember putting in an `init` function to serve a similar purpose.